test/dashboard-e2e/config.ts

```diff
@@ -1,5 +1,9 @@
+import dotenv from 'dotenv';
```

Comment:
Is the `dotenv` required here? I've seen that you deleted it in the last Cloud PR.

Reply:
I deleted it from playwright config (moved it to config.ts where ENVs are currently loaded)

Reply:
I moved it from `playwright.config.ts` to `config.ts` because otherwise ENVs from `.env` file weren't loaded correctly

Reply:
Ahh, that's right - `playwright.config.ts` is loading the `config.ts` before adjusting the envs 👍